MAINTAINERS.MD

```diff
@@ -1,3 +1,7 @@
-| Org                    | Name                                      | GitHub Username        |
-| -----------------------| ------------------------------------------| -----------------------|
-| §company§ | §name§ | §github_username§ |
+| Org                    | Name                                      | Role | GitHub Username        |
+| -----------------------| ------------------------------------------| ---| -----------------------|
+| Ericsson | Thorsten Lohmar | Condeowner & Maintainer | @tlohmar |
+| Nokia | Peter Kovacs | Condeowner & Maintainer | @PeterKovacs2 |
+| Telefonica| Jorge Garcia Hospital | Condeowner & Maintainer | @jgarciahospital |
+| Vodafone | Eric Murray | Condeowner | @eric-murray |
+| Vodafone | Steve Vickers | Maintainer | @SteveV-Vodafone |
```

Comment:
Please stay with the given format of the files. Maintainers go here, Codeowners have to go into the CODEOWNERS file. The roles are then implicit given.

```suggestion
| Org                    | Name                                      | GitHub Username        |
| -----------------------| -----------------------------------------| -----------------------|
| Ericsson | Thorsten Lohmar | @tlohmar |
| Nokia | Peter Kovacs | @PeterKovacs2 |
| Telefonica| Jorge Garcia Hospital |  @jgarciahospital |
| Vodafone | Eric Murray | @eric-murray |
| Vodafone | Steve Vickers |  @SteveV-Vodafone |
```